server/memphis_cloud.go

```diff
@@ -1363,6 +1369,25 @@ func (umh UserMgmtHandler) RemoveUser(c *gin.Context) {
 		return
 	}
 
+	deleteRequest := models.DeleteUserRequest{
```

Comment:
Wrap in a function sendCacheUpdate or something and don't return 500 to the user in case the cache didn't update